src/public/js/main.js

```diff
@@ -6,6 +6,11 @@
 
   function main(CONSTANTS) {
     var socket = io();
+    var uname = '';
+    var n = 255;
+    var k = 245;
+
+    $('#uname').val('delete this and enter your user name here');
```

Comment:
I think I should definitely look in to UI for my next job! Clearly it is where my skill set lies ;)